projects/project-player-library/src/lib/pages/details-page/details-page.component.ts

```diff
@@ -207,7 +207,7 @@ export class DetailsPageComponent implements OnInit {
         options.push(actions.CERTIFICATE_ACTION)
       }
     }
-    if(this.projectDetails.isDownload){
+    if(this.projectDetails.isDownload && !this.submitted){
```

Comment:
Keep the previous logic itself
But put this if block after line 202

Reply:
okay